Mage.Sets/src/mage/sets/JuniorSuperSeries.java

```diff
@@ -0,0 +1,42 @@
+package mage.sets;
+
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * https://scryfall.com/sets/psus
+ */
+public class JuniorSuperSeries extends ExpansionSet {
+
+    private static final JuniorSuperSeries instance = new JuniorSuperSeries();
+
+    public static JuniorSuperSeries getInstance() {
+        return instance;
+    }
+
+    private JuniorSuperSeries() {
+        super("Junior Super Series", "PSUS", ExpansionSet.buildDate(2008, 1, 1), SetType.PROMOTIONAL);
+        this.hasBoosters = false;
+        this.hasBasicLands = false;
+
+        cards.add(new SetCardInfo("City of Brass", 6, Rarity.SPECIAL, mage.cards.c.CityOfBrass.class));
```

Comment:
Why it special rarity? It's a rare in inscryfall: https://scryfall.com/card/psus/6/city-of-brass

Reply:
Because Scryfall only categorizes into (Common, Uncommon, Rare, Mythic rare).

- https://scryfall.com/card/vma/2/time-walk is marked as Mythic, while in XMage it is `Rarity.BONUS`.
- https://scryfall.com/card/eld/266/forest is a Common on Scryfall and a `Rarity.LAND` in XMage.

Reply:
All basic land must be LAND type. SPECIAL type uses only for two ravnika set to find special card:
![shot_200117_221601](https://user-images.githubusercontent.com/8344157/72635884-4af9d580-3977-11ea-85b8-b0a9dc0e6162.png)

I don't like SPECIAL -- it can be removed in the feature to support scryfall compatible (scryfall rarity uses real card's rarity, but official gatherer uses rarity as "booster place"). Search `public List<CardInfo> getSpecialLand()` for special cards usage in booster without SPECIAL rarity type (as example).

So:
1. Don't change rarity types in ravnika sets (it's can be next task to remove outdated SPECIAL type);
2. For all new sets add real rarity from scryfall (and LAND for basic lands), not special;
3. You can remove SPECIAL rarity from old promotional sets and make it same with scryfall;

Reply:
Ok, if that's the direction, I'll do so.
And I will change all SPECIAL cards I added in the commits of thid. PR.

Reply:
Replaced the SPECIAL rarity with RARE in all sets added by me (as listed on Scryfall).

I’ve checked them against Scryfall individually again, to make sure the rarity matches.
I have rebased and squashed the individual fix commits, the newly pushed commits don’t contain SPECIAL any more.

Side note: This’ll reduce my work, as I can just let my automation script run without the need to change rarity to SPECIAL any more ;) )

Should be resolved.